src/UI/Icons.elm

```diff
@@ -30,6 +34,40 @@ import Element exposing (..)
 import Html
 import Html.Attributes as HtmlAttr
 import UI.Attributes exposing (title)
+import UI.RenderConfig exposing (RenderConfig)
+
+
+
+-- NEW
+
+
+type Icon
+    = Toggle
+    | Close
+
+
+toggle : Icon
```

Comment:
Why this is called toggle as it renders a map? 

I'm aware that we will have to refactor everything here, but the icons should receive a text as a hint, accessibility purposes  :D

(Mental note, we should rename this component to UI.Icon)

Reply:
I also thought about that, but as it would add a giant diff to the review I left it to be commited at the end